composer.json

```diff
@@ -34,7 +34,7 @@
         "laminas-api-tools/api-tools-api-problem": "^1.2.2",
         "laminas-api-tools/api-tools-doctrine": "^2.1",
         "laminas-api-tools/api-tools-hal": "^1.4.2",
-        "laminas/laminas-hydrator": "^1.1 || ^2.2.1 || ^3.0",
+        "laminas/laminas-hydrator": "^1.1 || ^2.2.1 || ^3.0 || ^4.0",
```

Comment:
May I suggest dropping `^1` and `^2`?

Reply:
`doctrine/doctrine-module` should also be bumped to `^4.1.0` **only** (dropping old versions)

Reply:
I think this makes total sense, will do now.